packages/builders/src/interactions/slashCommands/Assertions.ts

```diff
@@ -5,6 +5,11 @@ import type { ToAPIApplicationCommandOptions } from './SlashCommandBuilder.js';
 import type { SlashCommandSubcommandBuilder, SlashCommandSubcommandGroupBuilder } from './SlashCommandSubcommands.js';
 import type { ApplicationCommandOptionBase } from './mixins/ApplicationCommandOptionBase.js';
 
+export class MissingRequiredParameterError extends Error {
+	public constructor(missingParameter: string) {
+		super(`Required parameter "${missingParameter}" is missing`);
+	}
+}
```

Comment:
You don't need to create this (and make the function return non-shapeshift errors), you can use the [`MissingPropertyError`](https://github.com/sapphiredev/shapeshift/blob/main/src/lib/errors/MissingPropertyError.ts) error class from `@sapphire/shapeshift`, which is [exported](https://github.com/sapphiredev/shapeshift/blob/main/src/index.ts#L12).

Using the aforementioned class:
```ts
import { MissingPropertyError } from '@sapphire/shapeshift';

throw new MissingPropertyError('name');
```